src/mixer/basetrackplayer.cpp

```diff
@@ -192,6 +192,13 @@ BaseTrackPlayerImpl::BaseTrackPlayerImpl(
 
     m_pRateRatio = make_parented<ControlProxy>(getGroup(), "rate_ratio", this);
     m_pPitchAdjust = make_parented<ControlProxy>(getGroup(), "pitch_adjust", this);
+
+    m_pUpdateReplayGainFromDeckGain = std::make_unique<ControlPushButton>(
+            ConfigKey(getGroup(), "update_replaygain_from_pregain"));
```

Comment:
This is quite a verbose name for a ControlObject... how about just `update_replaygain`... or `set_replaygain`? `store_replaygain`?

Reply:
I like the name. Our other CO names are often extremely ambiguous (so this PR introduces inconsistency 😅). You suggestions sound like I can set a the replaygain value directly, not like a button CO that takes only 1 and 0.

Reply:
yeah I don't know how to make it shorter and include the ideas of both the deck pregain and the track replaygain.  "apply_pregain_to_replaygain" ?

Reply:
I don't see how to remove any more words without making it confusing, and I don't think we have any limitations on CO name length

Reply:
I haven't thought of any better name, so... :shrug: I won't object to merging this over finding a better name.